bin/metrics-jenkins.rb

```diff
@@ -71,10 +71,33 @@ class JenkinsMetrics < Sensu::Plugin::Metric::CLI::Graphite
          description: 'Enabling https connections',
          default: false
 
+  option :timeout,
+         short: '-t SECONDS',
+         long: '--timeout SECONDS',
+         description: 'Timeout for REST request',
+         proc: proc(&:to_i),
+         default: 10
+
+  option :disable_request_duration,
```

Comment:
I'd prefer using a name like `report_request_duration` as this is more straightforward and does not require the double negation.

Reply:
Sounds good, will change for `--[no-]report-request-duration`.